java/statestore/src/main/java/sleeper/statestore/dynamodb/DynamoDBFileInfoFormat.java

```diff
@@ -122,4 +144,12 @@ FileInfo getFileInfoFromAttributeValues(Map<String, AttributeValue> item) throws
         }
         return fileInfoBuilder.build();
     }
+
+    static String getActiveFileSortKey(FileInfo fileInfo) {
+        return fileInfo.getPartitionId() + DELIMITER + fileInfo.getFilename();
+    }
+
+    private static String getFilenameFromSortKey(Map<String, AttributeValue> item) {
+        return item.get(PARTITION_ID_AND_FILENAME).getS().split(Pattern.quote(DELIMITER))[1];
```

Comment:
As Pattern.quote(DELIMITER) is constant, can we make it a final static variable?